lib/p2p/Peer.ts

```diff
@@ -496,15 +487,15 @@ class Peer extends EventEmitter {
 
       switch (err.type) {
         case ParserErrorType.InvalidPacket:
-          this.logger.warn(`parser: invalid peer packet: ${err.payload}`);
+          this.logger.error(`parser: invalid peer packet: ${err.payload}`);
           this.emit('reputation', ReputationEvent.InvalidPacket);
           break;
         case ParserErrorType.UnknownPacketType:
-          this.logger.warn(`parser: unknown peer packet type: ${err.payload}`);
+          this.logger.error(`parser: unknown peer packet type: ${err.payload}`);
```

Comment:
i'd say that `error` is for problems in the local node, while here it's most likely detecting a problem in the peer node.

Reply:
OK, that makes sense. I'll change it back.